src/components/AddTaskCard/AddTaskCard.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import { sanitizeText } from "src/utils/sanitizeText";
+import "./AddTaskCard.scss";
+
+function AddTaskCard({ onCreateTask }) {
+  const [taskTitle, setTaskTitle] = useState("");
+  const sanitizedTaskTitle = sanitizeText(taskTitle);
+  const isTaskButtonDisabled = sanitizedTaskTitle === "";
+
+  function handleChange(event) {
+    setTaskTitle(event.target.value);
+  }
+
+  function handleAddTask() {
+    onCreateTask(sanitizedTaskTitle);
+  }
+
+  return (
+    <div>
+      <textarea
+        className="task__textarea"
+        autoFocus
+        value={taskTitle}
+        onChange={handleChange}
+      />
+
+      <button
+        className={`task__save-button ${
```

Comment:
use this npm package here https://www.npmjs.com/package/classnames  
```suggestion
        className={classnames('task__save-button', {
        "task__save-button-disabled ": isTaskButtonDisabled
        }
```